cumulus/polkadot-parachain/src/command.rs

```diff
@@ -655,9 +689,9 @@ pub fn run() -> Result<()> {
 					SubstrateCli::create_configuration(&polkadot_cli, &polkadot_cli, tokio_handle)
 						.map_err(|err| format!("Relay chain argument error: {}", err))?;
 
-				info!("Parachain id: {:?}", id);
-				info!("Parachain Account: {}", parachain_account);
-				info!("Is collating: {}", if config.role.is_authority() { "yes" } else { "no" });
+				info!("🪪 Parachain id: {:?}", id);
+				info!("🧾 Parachain Account: {}", parachain_account);
```

Comment:
```suggestion
```

Not sure we need this anymore.

Reply:
Why not though? it seems like a nice user friendly addition to me. As a noob, it would be nice to see it.

Reply:
I also think it's nice to have it.